libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -300,7 +308,7 @@ void CodeTransform::operator()(Identifier const& _identifier)
 		{
 			// TODO: opportunity for optimization: Do not DUP if this is the last reference
 			// to the top most element of the stack
-			if (int heightDiff = variableHeightDiff(_var, _identifier.name, false))
+			if (auto heightDiff = static_cast<size_t>(variableHeightDiff(_var, _identifier.name, false)))
```

Comment:
`variableHeightDiff()` now returns `size_t`.
```suggestion
			if (size_t heightDiff = variableHeightDiff(_var, _identifier.name, false))
```